package.json

```diff
@@ -98,8 +100,6 @@
     "webpack": "^5.72",
     "webpack-cli": "^4.9.1",
```

Comment:
Also change it here. This is used in the tests - and we test both the upper and highest matching versions.

Reply:
I wasn't sure if it was needed here also.
Thanks, it's done.